quinn-udp/build.rs

```diff
@@ -0,0 +1,22 @@
+#![cfg(any(target_os = "macos", target_os = "ios"))]
+
+use std::env;
+use std::path::PathBuf;
+
+fn main() {
+    // Generate the bindings for Apple's private `recvmsg_x` from
+    // https://github.com/apple-oss-distributions/xnu/blob/main/bsd/sys/socket.h.
+    let bindings = bindgen::Builder::default()
```

Comment:
Bindgen is a pretty big and unstable dependency. Would it be simpler to commit the generated bindings?